src/components/App.js

```diff
@@ -246,26 +249,35 @@ class App extends Component {
 
         <div className="bottom-bar">
           <div className="bottom-left">
-            <div id="previous" onClick={this.previousTrack} title="Play Last">
-              <i className="fa fa-backward fa-2x" />
+            <div title="Play Previous">
+              <TiArrowLeftThick
+                onClick={this.previousTrack}
```

Comment:
Don't add the click handler on the actual icon. This has the same implications of adding it to a div.

Reply:
Okay, changing.